test/tools/unified-spec-runner/operations.ts

```diff
@@ -497,7 +497,7 @@ operations.set('replaceOne', async ({ entities, operation }) => {
 
 operations.set('startTransaction', async ({ entities, operation }) => {
   const session = entities.getEntity('session', operation.object);
-  session.startTransaction();
+  session.startTransaction(operation.arguments);
```

Comment:
One issue this update exposed was the args weren't getting passed to 'startTransaction`. I've also updated the test changes in this PR to address my comments in https://github.com/mongodb/specifications/pull/1502